docs/querying/sql-aggregations.md

```diff
@@ -139,6 +139,16 @@ Load the [DataSketches extension](../development/extensions-core/datasketches-ex
 |`DS_QUANTILES_SKETCH(expr, [k])`|Creates a [Quantiles sketch](../development/extensions-core/datasketches-quantiles.md) on the values of `expr`, which can be a regular column or a column containing quantiles sketches. The `k` parameter is described in the Quantiles sketch documentation.<br/><br/>See the [known issue](sql-translation.md#approximations) with this function.|`'0'` (STRING)|
 
 
+### Tuple sketch functions
+
+Load the [DataSketches extension](../development/extensions-core/datasketches-extension.md) to use the following functions.
+
+|Function|Notes|Default|
+|--------|-----|-------|
+|`DS_TUPLE_DOUBLES(expr, [nominalEntries])`|Creates a [Tuple sketch](../development/extensions-core/datasketches-tuple.md) on the values of `expr` which is a column containing Tuple sketches which contain an array of double values as their Summary Objects. The `nominalEntries` override parameter is optional and described in the Tuple sketch documentation.
```

Comment:
I've noticed you started including "Summary Objects" as part of the description. What does this mean?

I thought the last wording was clearer:

> Creates a Tuple sketch on the values of `expr` which is a column containing Tuple sketches.

It seems that all the Tuple sketches are also ArrayofDoublesSketch objects. https://druid.apache.org/docs/latest/development/extensions-core/datasketches-tuple.html

Reply:
I felt adding that information to the descriptions would help clarify things for those familiar with Tuple Data Sketches.

FYI, Summary Objects are explained in the Data Sketches documentation: https://datasketches.apache.org/docs/Tuple/TupleOverview.html

ArrayOfDoublesSketch is a specialized Tuple Sketch whose Summary Object is an array of double values.

Reply:
Thanks for the explanation! The doc changes overall look good to me.